app/controllers/employees_controller.rb

```diff
@@ -1,7 +1,6 @@
 class EmployeesController < ApplicationController
-
+  before_action :authenticate_user!
```

Comment:
add this in the application controller as you need authentication in every controller